docs/settings.md

```diff
@@ -189,16 +189,14 @@ limit resolutions to those present on that first index (`first-match`). This pre
 "dependency confusion" attacks, whereby an attack can upload a malicious package under the
 same name to a secondary.
 
-Possible values:
+**Possible values**:
 
-- `"first-index"`:        Only use results from the first index that returns a match for a given package name.
-- `"unsafe-first-match"`: Search for every package name across all indexes, exhausting the versions from the first index before moving on to the next.
-- `"unsafe-best-match"`:  Search for every package name across all indexes, preferring the "best" version found. If a package version is in multiple indexes, only look at the entry for the first index.
```

Comment:
We don't have to worry about the alignment here because we're not rendering to a terminal.